packages/terra-core-docs/src/terra-dev-site/doc/search-field/example/SearchFieldBlock.jsx

```diff
@@ -1,18 +1,11 @@
 import React, { useState } from 'react';
 import SearchField from 'terra-search-field';
 
-const INVALID_MESSAGE = 'The default minimum search length is 2.';
-
 const SearchFieldBlock = () => {
-  const [searchText, setSearchText] = useState('');
-
-  const message = searchText.length >= 2 ? `Search text: ${searchText}` : INVALID_MESSAGE;
+  const [setSearchText] = useState('');
```

Comment:
what is the purpose of this state variable?

Reply:
Answered on Sydney's comment: https://github.com/cerner/terra-core/pull/3849#discussion_r1291623111